### DISCUSSION TITLE
Possible to hide or remove some emotes from input autocompletion?

### DISCUSSION THREAD

**Discussion by Coldblackice** [Chatterino/chatterino2#3305] [Q&A] [answered]
Is there a way to hide or remove some emotes from input (tab) autocompletion? Given this is for text/input entry, "_Ignores_" wouldn't have an effect here.

Trying to remove some emote options from autocomplete so as not to have to tab through 50 different "Kapp_"s to get the one I use, for example.

**Comment by pajlada** [comment, marked as answer]
There is no way to do it right now, no. But I know others have expressed the same want. See https://github.com/Chatterino/chatterino2/issues/2438

**Reply by Coldblackice** [reply]
Ah k, thanks!
